io.github.diamondburned.gtkcord4.yml

```diff
@@ -0,0 +1,235 @@
+id: io.github.diamondburned.gtkcord4
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: gtkcord4
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --own-name=com.github.diamondburned.gtkcord4
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-run/keyring
+  - --device=dri
+
+modules:
+  - name: gtkcord4
+    build-commands:
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/golang/bin
+      env:
+        GOBIN: /app/bin
+        DESKTOP_ID: com.github.diamondburned.gtkcord4
```

Comment:
are you sure this is right?

Reply:
I switched to using the .desktop file from upstream https://github.com/flathub/flathub/pull/3857#issuecomment-1406728014 which in the source is named `com.github.diamondburned.gtkcord4.desktop`. The build-commands edits this .desktop file and renames it to use $FLATPAK_ID. I chose a variable because the rdns name is lengthy, and written more than once.

Reply:
`rename-dekstop-file` is probably what you want instead.

Reply:
`DESKTOP_ID` is now removed